src/wx/gfxviewers.cpp

```diff
@@ -1582,6 +1582,44 @@ class TileViewer : public GfxViewer {
         }
     }
 
+    void SaveGBATile(wxCommandEvent& ev)
+    {
+        (void)ev; // unused params
+        GameArea* panel = wxGetApp().frame->GetPanel();
+        wxString bmp_save_dir = wxGetApp().frame->GetGamePath(gopts.scrshot_dir);
+        // no attempt is made here to translate the dialog type name
+        // it's just a suggested name, anyway
+        wxString def_name = panel->game_name() + wxT('-') + dname;
+        def_name.resize(def_name.size() - 6); // strlen("Viewer")
+
+        if (captureFormat)
+            def_name += wxT(".bmp");
+        else
+            def_name += wxT(".png");
+
+        wxFileDialog dlg(GetGrandParent(), _("Select output file"), bmp_save_dir, def_name,
+            _("PNG images|*.png|BMP images|*.bmp"), wxFD_SAVE | wxFD_OVERWRITE_PROMPT);
+        dlg.SetFilterIndex(captureFormat);
+        int ret = dlg.ShowModal();
+        bmp_save_dir = dlg.GetDirectory();
+
+        if (ret != wxID_OK)
+            return;
+
+        wxBitmap obmp = gv->bm->GetSubBitmap(wxRect(0, 0, gv->bmh, gv->bmh));
```

Comment:
It should be `gv->bmw, gv->bmh` instead of `gv->bmh, gv->bmh`.

Reply:
When you do a `rebase -i` you can stop to amend a commit.